packages/ember-runtime/lib/mixins/array.js

```diff
@@ -964,17 +959,7 @@ const ArrayMixin = Mixin.create(Enumerable, {
     @public
   */
   invoke(methodName, ...args) {
-    let ret = A();
-
-    this.forEach((x, idx) => {
-      let method = x && x[methodName];
-
-      if ('function' === typeof method) {
-        ret[idx] = args.length ? method.apply(x, args) : x[methodName]();
-      }
-    }, this);
-
-    return ret;
+    return this.map(item => tryInvoke(item, methodName, args));
```

Comment:
This change breaks the API of `invoke`, which now no longer returns an `Ember.A` when prototype extensions are disabled.

Reply:
is wrapping it with `Ember.A` ok ? or revert will help ?

Reply:
I'm going to revert just this part, and add some tests

Reply:
https://github.com/emberjs/ember.js/pull/16785